### PR TITLE
Plain text templates

### PR DESCRIPTION
**Pull request by phamann**
This adds the templates and rendering logic for plain text email.

- Refactor the email data structure to acomodate plain text formats
- Consolidate `generate-html` & `generate-text` libs into a generic `render` function
- Add `-plain` view templates
- Refactor tests

### CODE REVIEW

File: functions/create-email/lib/email/generate-email.js [hunk 1]
```diff
@@ -1,29 +1,75 @@
 'use strict';
-const html = require('./generate-html');
-const text = require('./generate-text');
+const render = require('./render');
 
 const emails = {
-	daily: {
+	'daily': {
```
Comment (laurieboyes) [inline functions/create-email/lib/email/generate-email.js]: Are 'daily' and 'email-daily-digest' both still valid or should the latter have succeeded the former?

Comment (phamann) [inline functions/create-email/lib/email/generate-email.js]: The latter is the correct, but have left `daily` in for the interim for testing purposes until someone gets around to normalising the data structure across the functions/db.

**Comment by laurieboyes** [comment]
nice 👍

File: functions/create-email/lib/email/generate-email.js (hunk 1 above)
Comment (laurieboyes) [inline functions/create-email/lib/email/generate-email.js]: 👍

**Pull request merged**